apps/dot_ext/admin.py

```diff
@@ -19,3 +22,25 @@
 #
 #
 # admin.site.register(Application, ApplicationAdmin)
+
+
+class AccessTokenAdmin(admin.ModelAdmin):
+
+    list_display = ('get_user_username', 'get_app_name',)
+    search_fields = ('user__username', 'application__name',)
+    list_filter = ("user__username", "application__name")
+
+    def get_app_name(self, obj):
+        return obj.application.name
+
+    get_app_name.admin_order_field = "name"
+    get_app_name.short_description = "Application Name"
+
+    def get_user_username(self, obj):
+        return obj.user.username
+
+    get_user_username.admin_order_field = "username"
+    get_user_username.short_description = "User Name"
+
+admin.site.unregister(AccessToken)
```

Comment:
Why unregister `AccessToken` and the regeister it?

Reply:
@whytheplatypus We have to do that to swap out default behavior.  @ekivemark I'll work on fixing this as you asked.

Reply:
@ekivemark @whytheplatypus   We didn't need to unregister / register.  Instead I just created a new Admin for the same model by defining a proxy model.